src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color.GetColor getRandomColor() {
+        int index = new Random().nextInt(Color.GetColor.values().length);
```

Comment:
make Random class-level variable to avoid creating Random object each time you are calling getRandomColor